backend/src/modules/transactions/schemas/category.schema.ts

```diff
@@ -0,0 +1,7 @@
+import * as mongoose from 'mongoose';
+
+export const CategorySchema = new mongoose.Schema({
+  Name: String,
+  Icon: String,
+  _id: String,
```

Comment:
Dar sitas neresolvintas ;D

Reply:
> Sitas man atrodo yra nereikalingas: https://mongoosejs.com/docs/guide.html#_id

Reply:
Kazkodel pasalinus id lauka is schemos jis grazina objektus isvis be id ir niekaip nerandu kodel